tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -204,24 +204,44 @@ class KZG {
 
   template <typename ScalarContainer>
   [[nodiscard]] bool Commit(const ScalarContainer& v, Commitment* out) const {
+#if TACHYON_CUDA
+    if (msm_gpu_) {
+      return DoMSM(d_g1_powers_of_tau_, v, out);
+    }
+#endif
     return DoMSM(g1_powers_of_tau_, v, out);
```

Comment:
```suggestion
#if TACHYON_CUDA
    if (msm_gpu_) {
      return DoMSM(d_g1_powers_of_tau_, v, out);
    }
#else
    return DoMSM(g1_powers_of_tau_, v, out);
#endif    
```
Please change other parts as well.

Reply:
Oh this is intended, because MSM Gpu can't be disabled at any time, at this point, we want to compute it with CPU version.